tests/languages/javascript/number_feature.test

```diff
@@ -7,11 +7,21 @@
 0o571
 0xbabe
 0xBABE
+
 NaN
 Infinity
+
 123n
 0x123n
 
+1_000_000_000_000
+1_000_000.220_720
+0b0101_0110_0011_1000
+0x40_76_38_6A_73

```

Comment:
Could add a test for `0o12_34_56` for completeness’ sake (although I don’t think there’s a good use case for separators in such literals)

Reply:
https://github.com/tc39/proposal-numeric-separator/issues/44

Reply:
Added.